src/server/protocol.ts

```diff
@@ -2391,6 +2392,19 @@ export interface MapCodeResponse extends Response {
     body: readonly FileCodeEdits[];
 }
 
+export interface CopilotRelatedRequest extends FileRequest {
+    command: CommandTypes.CopilotRelated;
+    arguments: FileRequestArgs;
+}
+
+export interface CopilotRelatedItems {
+    relatedFiles: readonly string[];
+    traits: { name: string; value: string; }[];

```

Comment:
what is this supposed to be?

Reply:
Project-related information for copilot like module, lib, target, etc; not implemented in this PR, but coming soon.

Reply:
But do we know how the structure will look like. May be add this when we actually set?

Reply:
It's not going to change substantially, although it may add some optional properties -- I believe there's a PR from the C++ team to do so. I'm going to start returning some project information as traits quite soon and C# is already doing so. So I'd like to add it now, and I can add any optional properties that we use then.